src/registrar/views/domain.py

```diff
@@ -641,38 +639,6 @@ def form_valid(self, formset, **kwargs):
             return super().form_valid(formset)
 
 
-class DomainYourContactInformationView(DomainFormBaseView):
```

Comment:
Remove YourContactInformation page since this page references Submitter which we are deprecating in this PR. All Submitter data has been transferred to the new Creator field.